test/routes/preview.js

```diff
@@ -60,4 +60,85 @@ describe('POST /api/preview', function () {
       })
       .catch(error => done(error))
   })
+  // TODO: this test is skipped due to the issue described in https://github.com/CCALI/a2jdat/issues/100
+  it.skip('renders a message when there are no active templates', function (done) {
+    const fileDataUrl = path.join(__dirname, '..', 'data', 'DEV', 'guides', 'Guide1264')
+    request(app)
+      .post('/api/preview')
+      .set('Accept', 'text/html')
+      .send({
+        answers: '{}',
+        fileDataUrl
+      })
+      .expect('Content-Type', /html/)
+      .expect(200)
+      .then(response => {
+        const parsedHtml = parse(response.res.text)
+
+        const htmlTitles = parsedHtml.querySelectorAll('title')
+        assert.equal(htmlTitles.length, 1, 'the document has one title')
+        assert.equal(htmlTitles.toString(), '<title>A2J document preview</title>', 'the title is correct')
+
+        const htmlParagraphs = parsedHtml.querySelectorAll('p')
+        assert.equal(htmlParagraphs.length, 1, 'the document has one paragraph')
+        assert.equal(htmlParagraphs.toString(), '<p>No documents could be previewed because there are no active text templates.</p>', 'the message is correct')
+
+        done()
+      })
+      .catch(error => done(error))
+  })
+  // TODO: this test is skipped due to the issue described in https://github.com/CCALI/a2jdat/issues/100
+  it.skip('renders a message when the only active templates have conditional logic', function (done) {
+    const fileDataUrl = path.join(__dirname, '..', 'data', 'DEV', 'guides', 'Guide1265')
```

Comment:
It might be better to have your test guides as a package that gets put in the guides folder or to have the VIEWER_PATH key in config.json point to to your test data since coding the path may mask whether the relative path is working properly